pkg/commonvpc/connectivity_result.go

```diff
@@ -111,3 +113,12 @@ func (cr ConnectivityResultMap) Equal(other ConnectivityResultMap) bool {
 	}
 	return true
 }
+
+func (cr *ConnectivityResult) String() string {
```

Comment:
Why is this function needed? We print the grouping results, as external nodes are always printed

Reply:
used in ibm nacl analysis, later we will support also nacl for aws